src/k-provider/common/multi-request-builder.js

```diff
@@ -61,7 +61,7 @@ export class MultiRequestResult {
    * @memberof MultiRequestResult
    * @type {Object}
    */
-  results: Array<ServiceResult> = [];
+  results: Array<ServiceResult | string> = [];
```

Comment:
if success is false then add error info in `err` object, and only if parse succeeds then return the results as `ServiceResult`.
We can create a standard object 
```
{success: boolean, results: Array<ServiceResult>, err: errObject}
```

Reply:
remove this